system/os_library/exports.js

```diff
@@ -46,6 +46,9 @@ for (var i in lines)
 }
 
 var _path = "/Users/gabrielvalky/Downloads/gcc-arm-none-eabi-7-2018-q2-update/bin/";
+if (!require("fs").existsSync(_path)) {
```

Comment:
Makes this part work if the hardcoded path doesn't exist. Should keep your side working while making it work for us too.

Reply:
please remove the hardcoded path

Reply:
done